### DISCUSSION TITLE
Exception thrown: 'Microsoft.JSInterop.JSException' in System.Private.CoreLib.dll

### DISCUSSION THREAD

**Discussion by Naganathan** [dotnet/aspnetcore#56505] [General]
Our project created in Blazor with WPF hybrid using NET8.0. When invoking JSInterop calls its thrown below exception from an external codes but application running as expected without a problem. 

![image](https://github.com/dotnet/aspnetcore/assets/5599700/4025b1c6-2a79-452f-9662-c50876f93533)

may I know it will create any problem for the application? if there is no issues means can we ignore?

**Comment by SwartBergStudio** [comment]
I currently have the same issue on my project.  Every time I click on MudFileUpload it gives this error.  But on file new project the button works with the exact same framework and packages.  So trying to find out what is broken on my projects to fix it

**Reply by yuvaltir** [reply]
Hi @Mecerburn,
Have you been able to resolve this by any chance?
I noticed this happens to me after upgrading MudBlazor, a new project I created though has Sdk="Microsoft.NET.Sdk.BlazorWebAssembly" whereas the old one is Sdk="Microsoft.NET.Sdk.Web".
Cheers,
Yuval

**Reply by yuvaltir** [reply]
OK, nevermind, I just ceared the browser cache and it solved the issue...
